components/01-atoms/divider/divider.twig

```diff
@@ -0,0 +1,9 @@
+{% set divider__base_class = 'divider' %}
```

Comment:
I think there should be comments here for the available variables like we've done for projects in the past. Because this is the first component it might be good to establish this practice now instead of retro-actively updating things.

Reply:
Thanks for reminding me about this! Done.